Dockerfile

```diff
@@ -1,21 +1,10 @@
-FROM okteto/okteto:2.0.1 as okteto
-
-FROM golang:1.16 as message-builder
-RUN go env -w GO111MODULE=off
-RUN go get github.com/machinebox/graphql
-COPY message.go .
-RUN go build -o /message .
-RUN curl https://stedolan.github.io/jq/download/linux64/jq > /usr/bin/jq && chmod +x /usr/bin/jq
 
-FROM ruby:3-slim-buster
-
-RUN gem install octokit
+FROM okteto/okteto:2.0.1 as okteto
+FROM ghcr.io/okteto/deploy-preview:latest
```

Comment:
@pchico83 @jLopezbarb using the latest there's a risk that the base image is not compatible with the image running on the tag. would it be ok if we create the image when creating the release instead?

Reply:
another option (my favorite) is to move the base image to another repo

Reply:
+1 for moving the base image to a different repo.
@rberrelleza if you want, create an issue for it, and we will take care of adding it to the planning

Reply:
https://github.com/okteto/actions-base has the base dependencies now.